rs.py

```diff
@@ -44,4 +44,4 @@ def classify():
 
 if __name__ == '__main__':
     configure_service('resources/config/app_config.cfg')
```

Comment:
Im not sure if they contain anything sensitive, might have to ask Aaron, but if not would this just involve taking the `resources/` out of the `.gitignore` file?